### PR TITLE
List Group Index (Wearable): update example and style

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/936
[Problem] Group index example is at wrong place and its
	style are outdated according to guide.
[Solution] Update example and style.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/list/index.html [hunk 1]
```diff
@@ -23,6 +23,11 @@ <h2 class="ui-title">
 		</header>
 		<div class="ui-content">
 			<ul class="ui-listview">
+				<li>
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/presentation-views/list/index.html]: to keep consistency with guide please place it after Main Text Only

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 2]
```diff
@@ -53,36 +53,14 @@
...
 
 		&.ui-listview-divider, &.ui-li-group-index {
-			height: 33 * @unit_base;
-			min-height: 33 * @unit_base;
-			line-height: 31 * @unit_base;
-			background-color: transparent;
-			color: @color_list_divider_text;
-			display: flex;
-			flex-direction: row;
+			height: 32 * @px_base;
```
Comment (lslachciak) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: do we need to define both height and min-height?

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less [hunk 3]
```diff
@@ -163,36 +163,34 @@ input[type="radio"] {
...
+	label {
+		display: flex;
+		align-items: center;
+
+		input[type="checkbox"], input[type="radio"] {
+			position: relative;
+			top: 0;
+			left: 0;
+			transform: none;
+			min-width: 76 * @unit_base;
```
Comment (lslachciak) [inline src/css/profile/wearable/changeable/theme-circle/checkboxradio.less]: please use px_base as in other places

**Comment by lslachciak** [comment]
Generally PR works well, but I have some comments regarding code

File: examples/wearable/UIComponents/contents/presentation-views/list/index.html (hunk 1 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/index.html]: ok updated

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 2 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: li for ui-listview has defined both min-height and max-height (see above). Here we need to overwrite them. Changed to max-height to have correct size.

File: src/css/profile/wearable/changeable/theme-circle/checkboxradio.less (hunk 3 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/checkboxradio.less]: Done.

**Comment by grzegorz-czajkowski** [comment]
Created separated issue for wrong position of group index https://github.com/Samsung/TAU/issues/952

**Review by lslachciak — APPROVED** [review]

**Comment by lslachciak** [comment]
works fine now 😃

**Pull request merged**